src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -72,6 +72,7 @@ where
 {
     hummock_manager: HummockManagerRef<S>,
     compactor_manager: CompactorManagerRef,
+    compactor_selection_retry_interval_sec: u64,
```

Comment:
Shall we need a `CompactionSchedulerConfig` ? may be we can control `CompactionScheduler` more precisely. And more readable

Reply:
Sure. Can be done when there are more than one parameters.